frontend/src/component/changeRequest/ProjectChangeRequests/ChangeRequestsTabs/ChangeRequestsTabs.tsx

```diff
@@ -172,13 +173,16 @@ export const ChangeRequestsTabs = ({
         useSortBy
     );
 
-    useEffect(() => {
-        const hiddenColumns = [''];
-        if (isSmallScreen) {
-            hiddenColumns.push('createdBy', 'updatedAt');
-        }
-        setHiddenColumns(hiddenColumns);
-    }, [setHiddenColumns, isSmallScreen]);
+    useConditionallyHiddenColumns(
+        [
+            {
+                condition: isSmallScreen,
+                columns: ['createdBy'],
```

Comment:
Seems like `updatedAt` no longer exists. I removed it but want to call attention to it.